src/routes/likernft/book/store.ts

```diff
@@ -45,14 +56,17 @@ router.get('/list', jwtOptionalAuth('read:nftbook'), async (req, res, next) => {
         stock,
         shippingRates,
         defaultPaymentCurrency,
+        timestamp: timestamp.toMillis(),
       };
       if (req.user && req.user.wallet === wallet) {
         result.pendingNFTCount = pendingNFTCount;
         result.sold = sold;
       }
       return result;
     });
-    res.json({ list });
+    const nextKey = list.length < conditions.limit ? null : list[list.length - 1].timestamp;
+    res.set('Cache-Control', 'public, max-age=6, s-maxage=6, stale-if-error=60');
```

Comment:
```suggestion
    res.set('Cache-Control', 'public, max-age=60, s-maxage=60, stale-if-error=600');
```
This is not block related, we can have longer cache time

Reply:
On second thought, we need to set `private` for `isAuthorized`